docs/manual/shared_memory.rst

```diff
@@ -63,7 +64,11 @@ SubQueueCapacity, SubHistoryRequest and PubHistoryCapacity can be optionally set
 SubQueueCapacity controls how many samples a reader using shared memory can hold before the least recent is discarded.
 PubHistoryCapacity defines how many samples a shared memory writer will keep to send to late-joining subscribers.
 SubHistoryRequest is the number of samples a late-joining reader will request from a writer (this can be at most 
-as many as were send and at most PubHistoryCapacity).   
+as many as were send and at most PubHistoryCapacity).
```

Comment:
grammar correction: perfect tense of "to send" is "sent", change

Reply:
You are right, will change.